### PR TITLE
Data resource

### PR DESCRIPTION
**Pull request by delrueba**

### CODE REVIEW

File: src/actions/fetch-data-resource.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      method: 'post',
+      url: 'https://qa.stad.gent/sparql',
+      data: formData,
+    })
+  }
+  catch (e) {
+    throw e
+  }
+
+  response = response.data.results.bindings
```
Comment (hstandaert) [inline src/actions/fetch-data-resource.js]: Did you test this? Isn't this possibly going to throw an error? 🤔

Comment (delrueba) [reply src/actions/fetch-data-resource.js]: If the call was successful, data.result.bindings is always present.
Anyway, errors are catched in the calling function.
@nillusius?

File: src/actions/fetch-data-resource.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+  try {
+    response = await instance({
+      method: 'post',
+      url: 'https://qa.stad.gent/sparql',
+      data: formData,
+    })
+  }
+  catch (e) {
+    throw e
```
Comment (hstandaert) [inline src/actions/fetch-data-resource.js]: On second thoughts: this is kind of useless. Catching an error provides you with the possibility of handling the error in a custom way. Just throwing that same error again has the same outcome as just leaving out the `try...catch` statement

**Comment by delrueba** [comment]
I think we're not supposed to combine text with topic links. At least, that's what I get from the designs.

Also: if it's a collection of links, 'read more' is not correct (as there is nothing more to 'read')

Text: read more
collection of links: show more

**Pull request merged**